gateway/gateway.go

```diff
@@ -1380,24 +1372,24 @@ func (gateway *HandleT) StartWebHandler(ctx context.Context) error {
 
 	gateway.logger.Infof("Starting in %d", webPort)
 	srvMux := mux.NewRouter()
-	srvMux.Use(headerMiddleware)
-	srvMux.HandleFunc("/v1/batch", gateway.stat(gateway.webBatchHandler)).Methods("POST")
-	srvMux.HandleFunc("/v1/identify", gateway.stat(gateway.webIdentifyHandler)).Methods("POST")
-	srvMux.HandleFunc("/v1/track", gateway.stat(gateway.webTrackHandler)).Methods("POST")
-	srvMux.HandleFunc("/v1/page", gateway.stat(gateway.webPageHandler)).Methods("POST")
-	srvMux.HandleFunc("/v1/screen", gateway.stat(gateway.webScreenHandler)).Methods("POST")
-	srvMux.HandleFunc("/v1/alias", gateway.stat(gateway.webAliasHandler)).Methods("POST")
-	srvMux.HandleFunc("/v1/merge", gateway.stat(gateway.webMergeHandler)).Methods("POST")
-	srvMux.HandleFunc("/v1/group", gateway.stat(gateway.webGroupHandler)).Methods("POST")
+	srvMux.Use(gateway.statMiddleware, gateway.maxActiveClientMiddleware, headerMiddleware)
```

Comment:
@chandumlg with this change we are enabling stats collection for `/schemas` endpoints as well, I don't see any problem with that, on the contrary I believe it is desirable to collect metrics for all possible paths. Can you confirm?

Reply:
Yeah no problem. Let's enable.